resources/views/site/pilot_guide_atl.blade.php

```diff
@@ -70,51 +70,56 @@ function fetchBadge($pos) {
+                            <li>Concourse D:
                                 <ul>
-                                    <li>Gates D1/D1A/D3: Frontier</li>
-                                    <li>Gates D2/D4: Spirit</li>
-                                    <li>Gates D21/D23/D24/D25: American</li>
-                                    <li>All other gates: Delta connection</li>
+                                    <li>D1/D1A/D3: Frontier & Alaska [Common Use]</li>
+                                    <li>D2/D4: Spirit [Common Use]</li>
+                                    <li>D5 - D16: Delta Air Lines & Delta Connection</li>
+                                    <li>D21, D23, D25, D27, D29: American Airlines</li>
+                                    <li>D24, D26, D28: Delta Air Lines & Delta Connection</li>
+                                    <li>D30 - D46: Delta Connection</li>
                                 </ul>
                             </li>
                         </ul>
                     </div>
                     <div class="col">
                         <ul>
-                            <li>Concourse E
+                            <li>Concourse E:
                                 <ul>
-                                    <li>Gates E1/E3: Spirit</li>
-                                    <li>Gate E2: Frontier</li>
-                                    <li>Gate E32: JetBlue</li>
-                                    <li>Gate E34: United</li>
-                                    <li>All other gates: Delta domestic/international</li>
+                                    <li>E1 - E6: Spirit/Frontier [Common Use]</li>
+                                    <li>E7 - E33: Delta Air Lines [Common Use]</li>
+                                    <li>E34 - E36: JetBlue, Delta Air Lines [Common Use]</li>
+                                    <li>E37: Southern Airways Express</li>
                                 </ul>
                             </li>
-                            <li>Concourse F
+                            <li>Concourse F:
                                 <ul>
-                                    <li>Gate F1: British Airways & Korean Air</li>
-                                    <li>Gate F2: Virgin Atlantic & Delta</li>
-                                    <li>Gate F3: Lufthansa, Turkish & Delta</li>
-                                    <li>Gates F4/F6/F8/F9/F10/F12: Delta</li>
-                                    <li>Gate F5: Qatar</li>
-                                    <li>Gate F7: KLM & Air France</li>
-                                    <li>Gate F14: Air Canada & Delta</li>
+                                    <li>F1: Korean Air & All Group VI (B747-8, A380) aircraft, Delta Air Lines [Common Use]</li>
+                                    <li>F2 - F14: Common Use</li>
+                                </ul>
+                            </li>
+                            <li>Additional notes:
```

Comment:
Move these additional notes down to a new row so they fit neatly under the two columns of gate information

Reply:
I'm sorry - this doesn't look good. Can you please move it back into the 2nd column under concourse f?